js/layout-javascript/agenda-code.js

```diff
@@ -1778,6 +1782,21 @@ DynamicList.prototype.animateDateBack = function($prevDateElement, prevDateEleme
   });
 };
 
+DynamicList.prototype.correctPosition = function() {
+  var windowWidth = window.innerWidth;
+  var number = parseInt(Fliplet.Navigate.query.dateIndex, 10);
+  var difference = number - this.activeSlideIndex;
+  var currentAgendaElement = this.$container.find('.agenda-list-day-holder.active');
+  var currentAgendaElementWidth = Math.floor(currentAgendaElement.outerWidth() * difference);
+
+  this.ANIMATION_SPEED = 0;
+  this.scrollValue = $($('.agenda-cards-wrapper')[0]).scrollLeft();
```

Comment:
There's no need to wrap it in `$()` again.

```suggestion
  this.scrollValue = $('.agenda-cards-wrapper').eq(0).scrollLeft();
```